docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,184 @@
+
+<br />
+
+### Creating vmTemplates with CSI-assisted cloning
+
+The next step is to create a `vmTemplate` resource that clones the PVC in the `vmo-golden-images` namespace. This is done through the `dataVolumeTemplates` section:
+
+```
+apiVersion: spectrocloud.com/v1
+kind: VmTemplate
+metadata:
+  name: ubuntu-2204
+spec:
+  dataVolumeTemplates:
+    - metadata:
+        name: ubuntu-2204
+      spec:
+        source:
+          pvc: 
+            name: template-ubuntu-2204
+            namespace: vmo-golden-images
+        pvc:
+          accessModes:
+            - ReadWriteMany
+          resources:
+            requests:
+              storage: 50Gi
+          volumeMode: Block      # or "Filesystem", depending on the CSI used
+  template:
+    < rest of template >
+```
+
+The `spec.dataVolumeTemplates.spec` section defines which source PVC to clone and what PVC to clone into. In the example above, the `source` section points to the `template-ubuntu-2204` PVC in the `vmo-golden-images` namespace. The `pvc` section creates a new target PVC with the specified `accessMode` and `volumeMode` characteristics. The size of the new PVC must be the same or larger as the source PVC.
+
+:::info
+To allow CSI-assisted cloning to work, there are a few [prerequisites](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/csi-cloning.md#prerequisites) that must be followed. Most importantly, the source and target PVC must share the same Storage Class and the same Volume Mode. If the requirements are not met, CDI will automatically fallback to the slower host-assisted cloning.
+
+To enable CSI-assisted cloning into larger PVCs than the source PVC, the Storage Class used must have `allowVolumeExpansion` set to `true`.
+:::
+
+<br />
+
+### Setting up RBAC permissions for cloning across namespaces
+
+When the CDI performs a cloning operation, it runs under the `default` serviceaccount in the namespace of the target PVC. When the source PVC is in a different namespace, required permissions must be given to the serviceaccount. Version 4.2.0 of the VMO pack does this automatically, you can specify the namespaces for which the permissions are configured through the `vmEnabledNamespaces` option in the pack:
+
+```
+charts:
+  virtual-machine-orchestrator:
+
+    # This namespace will be used to store golden images
+    goldenImagesNamespace: "vmo-golden-images"
+
+    # These namespaces will be created and set up to deploy VMs into
+    vmEnabledNamespaces:
+      - "default"
+      - "virtual-machines"
+```
+
+<br />
+
+### Defining StorageProfiles to ensure CSI-assisted cloning
+
+While the CDI contains an internal database of CSIs and their supported access modes, volume modes and CSI-assisted cloning support, it is still recommended to create a `StorageProfile` to force these parameters to what we want them to be. For example:
```

Comment:
```suggestion
While the CDI contains an internal database of CSIs and their supported access modes, volume modes and CSI-assisted cloning support, we recommend creating a `StorageProfile` to force these parameters to their desired values, as follows.
```

Reply:
We use [active voice](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1765933057/Spectro+Cloud+Internal+Style+Guide#Active-voice) and the first plural for recommendations.